reconcile/gitlab_housekeeping.py

```diff
@@ -0,0 +1,73 @@
+import logging
+
+from datetime import datetime, timedelta
+
+from utils.config import get_config
+from utils.gitlab_api import GitLabApi
+
+
+def get_housekeeping_gitlab_api():
+    config = get_config()
+
+    gitlab_config = config['gitlab']
+    server = gitlab_config['server']
+    token = gitlab_config['token']
+    project_id = gitlab_config['housekeeping']['project_id']
+
+    return GitLabApi(server, token, project_id=project_id, ssl_verify=False)
+
+
+def handle_stale_issues(dry_run, gl):
+    DATE_FORMAT = '%Y-%m-%dT%H:%M:%S.%fZ'
+    LABEL = 'stale'
+    DAYS = 30
+
+    issues = gl.get_issues(state='opened')
+    now = datetime.utcnow()
+    for issue in issues:
+        issue_iid = issue.attributes.get('iid')
+        issue_labels = issue.attributes.get('labels')
+        updated_at = issue.attributes.get('updated_at')
+        update_date = datetime.strptime(updated_at, DATE_FORMAT)
+
+        # if issue is over 30d old
```

Comment:
I would suggest we mark as stale at 15 days of no activity, and then close after 15 days of having a stale label

Reply:
my vote goes to 15 days as well :+1:

Reply:
updated to 15!